staging/src/k8s.io/apimachinery/pkg/util/diff/compare.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package diff
+
+import "time"
+
+var allowedNTPDiffInMilliSecond = uint64(5000)
```

Comment:
constant?

Reply:
There are unit tests that test with different value of allowedNTPDiffInMilliSecond. If it is constant, the test can not be done.